### PR TITLE
Adds a demo to show how a sound alert can be played when a prediction is complete

### PR DESCRIPTION
**Pull request by abidlabs**
Adds a demo (`sound_alert.py`) to show how a sound alert can be played when a prediction is complete.

CLoses: #2173

### CODE REVIEW

File: demo/gif_maker/run.py [hunk 1]
```diff
@@ -1,10 +1,8 @@
 import cv2
 import gradio as gr
-import tempfile
```
Comment (abidlabs) [inline demo/gif_maker/run.py:3]: Just removes some unused imports I noticed

**Comment by aliabd** [comment]
Nice @abidlabs 

I'm seeing a console error on the demo, even though it runs as expected

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'forEach')
    at $ (index.b57b8f62.js:55:2384)
```

Also, consider adding the demo to the docstrings where applicable (maybe at blocks?) or to the demos page. Otherwise LGTM

**Comment by abidlabs** [comment]
Thanks for the review @aliabd! Will address these

**Review by freddyaboulton — APPROVED** [review]
Looks good to me @abidlabs!

**Comment by dawoodkhan82** [comment]
nice demo!

**Review by dawoodkhan82 — APPROVED** [review]

**Comment by abidlabs** [comment]
Thanks for the review folks! @aliabd I believe the JS error is happening because this event trigger doesn't have any inputs or outputs. Gonna leave that for a future PR but I've gone ahead and added this demo the `Blocks` class.

**Pull request merged**